
---

string[] filenames
bool success
string error_message
